### DISCUSSION TITLE
Google Fonts

### DISCUSSION THREAD

**Discussion by Barcenilla00** [templaza/astroid-framework#640] [Q&A]
I have installed 2 Google Fonts fonts locally on my server.
Through the Astroid Framework template options I have used these fonts only. However, calls to Google Fonts continue to appear in the <head>.
How can I avoid it?
![image](https://github.com/templaza/astroid-framework/assets/22943057/e66a12ef-20d3-4be6-9d94-641a2a72fb72)

**Comment by ghost** [comment]
Hallo,
i I had a same problem [here](https://github.com/templaza/astroid-framework/discussions/601).

Have you checked that the font is set correctly for all widgets (if you use them)?

@sonvnn 
Would it perhaps be an improvement if you could completely deactivate the loading of Google fonts in the Astroid plugin?

**Comment by Barcenilla00** [comment]
Yes, that is the reason... I've read another post with the answer.
There's a problem with que widgets configuration.
Every time I go into the settings the font appears as "Google" instead of "Local" which is what I choose.
I select "local", I save it and, if I enter again, Google appears again.
However, it does set it to "local" because the links to Google Fonts no longer appear

**Comment by ghost** [comment]
Hello

Maybe you mean [this](https://github.com/templaza/astroid-framework/discussions/616) post?
This also comes from me because I stumbled upon it and had to spend many hours looking for the problem.
I had exactly the same problem I think

**Comment by Barcenilla00** [comment]
Right!

**Comment by Barcenilla00** [comment]
See also https://github.com/templaza/astroid-framework/discussions/601

**Comment by ghost** [comment]
Maybe Sonvnn can find a good solution.

I think it would be good if you could basically deactivate the loading of Google fonts in the Astroid plugin. or you could set it up there if you want to load fonts from Google Server or prefer locally. Just like with Fontawesome

**Comment by mac-joomla** [comment]
Would be nice, if you choose to use a google font, then astroid would automatically download the font and use ist locally. Outstanding 🥇
